### DISCUSSION TITLE
希望更新蔚蓝最终版

### DISCUSSION THREAD

**Discussion by liushengqi000** [Gamuxorg/bbs#238] [投递和请求游戏]
游戏名称：蔚蓝
游戏名称(英文)：Celeste
下载地址 :https://www2.yggtorrent.se/torrent/jeu-vid%C3%A9o/linux/559852-linux+celeste+1+3+1+2

蔚蓝增加了第九章，作者似乎暂无续作计划，后面应该不会有更新了。希望能更新到最终版。

唔，如果可以的话 baba is you也希望能得到更新。

**Comment by hyh19962008** [comment]
已更新
